src/wallet/rpc/wallet.cpp

```diff
@@ -235,7 +235,11 @@ static RPCHelpMan loadwallet()
                 },
                 RPCExamples{
                     HelpExampleCli("loadwallet", "\"test.dat\"")
+            + HelpExampleCli("loadwallet", "\"specialWallets/test2.dat\"")
+            + HelpExampleCli("loadwallet", "\"/Users/joe/specialWallets\"")
```

Comment:
Tried `createwallet` to observe that the current version creates a directory with the given name, containing `wallet.dat` (and `wallet.dat-journal` while loaded).

Suggestion: put the normal/current/simplest operation first, and also add some descriptions to command examples:
```suggestion
                    "\nLoad regular wallet with files under wallets/foo/:\n"
                    + HelpExampleCli("loadwallet", "\"foo\"") +
                    "\nLoad wallet using absolute path:\n"
                    + HelpExampleCli("loadwallet", "\"/Users/joe/specialWallet/\"")                    
                    "\nLoad legacy wallet.dat file directly under the wallets/ directory:\n"
                    + HelpExampleCli("loadwallet", "\"legacy.dat\"")
```

Reply:
Great suggestion- added!